src/constants/constants.js

```diff
@@ -414,6 +414,62 @@ export const ARCHIVE_UPDATE_RESPONSES_FOLDER = 'update_responses';
  */
 export const COMMAND_QUEUE_PARALLELISM = 100;
 
+/**
+ * @constant {object} HTTP_API_ROUTES -
+ *  HTTP API Routes with parameters
+ */
+export const HTTP_API_ROUTES = {
+    v0: {
+        publish: {
+            method: 'post',
+            path: '/publish',
+            options: { rateLimit: true },
+        },
+        update: {
+            method: 'post',
+            path: '/update',
+            options: { rateLimit: true },
+        },
+        query: {
+            method: 'post',
+            path: '/query',
+            options: {},
```

Comment:
i see that you just transformed the old code, but curious - why are we not rate limiting query?

Reply:
Probably because it's not a network protocol